src/services/prebid.js

```diff
@@ -18,10 +18,12 @@ export function queuePrebidCommand(fn) {
  * @param {function} cb - An optional callback function that should fire whenever the bidding has concluded.
  **/
 export function fetchPrebidBids(ad, code, timeout, info, prerender, cb = null) {
+  pbjs.addAdUnits(info);
```

Comment:
this was not setting the header bidding ad units, instead it was only passing the name or ID of the ad slot